conf/config.properties.example

```diff
@@ -768,6 +768,18 @@ client.cmpdreg.serverSettings.liveDesign.url=http://localhost:8010/ld-chem
 
 client.cmpdreg.serverSettings.maxStandardizationDisplay=20000
 
+# Configures how many compounds are sent to the external structure processing service per request.
+server.acas.externalStructureProcessingBatchSize=100
```

Comment:
Corresponding configurations wwereas added to roo a long time ago with this as these defaults. I'm adding them into this PR as the way Kubernetes configs works, they can only be overridden when they exist in a configuration example file.  So this allows us to actually set the values in K8s deployments.
 https://github.com/mcneilco/acas-roo-server/blob/1ae076d8b787af3a8fa6f03b1b201d923b925c24/src/main/java/com/labsynch/labseer/utils/PropertiesUtilServiceImpl.java#L886

https://github.com/mcneilco/acas-roo-server/blob/1ae076d8b787af3a8fa6f03b1b201d923b925c24/src/main/java/com/labsynch/labseer/utils/PropertiesUtilServiceImpl.java#L903